google/cloud/status_or.h

```diff
@@ -24,33 +24,52 @@ namespace google {
 namespace cloud {
 inline namespace GOOGLE_CLOUD_CPP_NS {
 /**
- * `StatusOr<T>` is used to return a value or an error status.
+ * `StatusOr<T>` represents either a usable `T` value or a `Status` object
```

Comment:
These are really cool, thanks!  I am missing the "brief" description though, typically Doxygen comments are formatted as:

```C++
/**
  * Brief description of the thing, one liner.
  *
  * Longer description of the thing, with comments and examples, and stuff.
  * Maybe multiple lines.
  */
```

The brief description shows up in any list describing things, for example this:

https://googleapis.github.io/google-cloud-cpp/latest/storage/annotated.html

Reply:
Thanks. Fixed, I think.